PyCalc.py

```diff
@@ -0,0 +1,62 @@
+from math import pi, e
+from re import compile, sub
+
+
+def throwException(warning: str) -> None:
+    """Функция для отображения ошибок на консоли, вместо прерывания цикла с помощью raise"""
+    print("stderr: {}".format(warning))
+
+
+def constantCoding(expression: str) -> 'str, None':
+    """Функция проверяет выражение на наличие лишних символов и заменяет константы(pi && e)
+        (временно для проверки регулярными выражениями) на pi >> $$, e >> @ """
+    if '@' in expression or '$' in expression:
+        return None
+    return sub(r'pi', '$$', sub(r'e', '@', expression))
+
+
+syntaxCheckers = {
+    'undefined symbols': compile(r'[^0-9.()*/\-+%@$]'),   # 0-9, '.', '(', ')', '*', '/', '-', '+', '%', '\s', '@', '$'
+    'repeating operations': compile(r'[+\-]{2,}|\.{2,}')  # '+', '-', '.'
+}
+
+
+def checkingExpression(expression: str) -> bool:
+    """Функция для проверки выражения на наличие других символов,
+        кроме: 0-9, '.', '(', ')', '*', '/', '-', '+', '%', '\s', '@', '$'.
+        Так же проверяется, не повторяются ли подряд '+', '-', '.' """
```

Comment:
```suggestion
    """
    Функция для проверки выражения на наличие других символов,
        кроме: 0-9, '.', '(', ')', '*', '/', '-', '+', '%', '\s', '@', '$'.
        Так же проверяется, не повторяются ли подряд '+', '-', '.' 
    """
```

Знехтував пунктом 3.4.2 
Роз’яснення повинні починатися з трьох лапок та містити тільки ці символи, так само і кінець роз’яснення повинно закінчуватись на три лапки в окремому рядку